Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -893,7 +917,9 @@ private Bucket NewBucket(GcsPath gcsPath, NewGcsBucketDynamicParameters dynamicP
         }
 
         private void ObsoleteBucketCache()
-        { }
+        {
+            BucketCache.Obsolete();
```

Comment:
How many times is `ObsoleteBucketCache` called? Given that the function name is 100% transparent, why not replace its callsites to simply `BucketCache.Obsolete`? YAGNI